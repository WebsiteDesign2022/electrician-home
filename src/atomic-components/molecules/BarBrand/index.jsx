import React from 'react';
import { Navbar } from 'react-bootstrap';

const BarBrand = ({ brandName }) => (
    <Navbar.Brand className="px-4 navbar-brand">
        {brandName}
    </Navbar.Brand>
)

export default BarBrand;
