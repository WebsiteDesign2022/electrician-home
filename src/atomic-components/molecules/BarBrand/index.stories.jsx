import React from 'react';

import BarBrand from '.';

export default {
    title: 'Electrician-Home/molecules/BarBrand',
    component: BarBrand,
    parameters: {
      // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
      layout: 'fullscreen',
    },
};

const Template = (args) => <BarBrand {...args} />;

export const Primary = Template.bind({});
Primary.args = {
    brandName: 'Electrician Home'
};

// Empty state unlikely.
