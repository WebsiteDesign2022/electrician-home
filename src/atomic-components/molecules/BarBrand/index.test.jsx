import TestRenderer from 'react-test-renderer';

import BarBrand from '.';

describe("BarBrand", () => {
    test("renders BarBrand", () => {
        const barBrand = TestRenderer.create(
            <BarBrand brandName="author mark" />
        );

        expect(barBrand.toJSON()).toMatchSnapshot();
    });
});
