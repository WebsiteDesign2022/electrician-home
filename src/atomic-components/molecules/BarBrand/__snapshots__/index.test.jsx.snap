// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarBrand renders BarBrand 1`] = `
<span
  className="px-4  navbar-brand"
>
  author mark
</span>
`;
